src/neuroglancer/sliceview/segmentation_renderlayer.ts

```diff
@@ -153,6 +157,17 @@ uint64_t getMappedObjectId() {
     alpha = uNotSelectedAlpha;
   }
   vec3 rgb = segmentColorHash(value);
+`;
+
+    // Override color for all highlighted segments.
+    fragmentMain += `
+  if(${this.hashTableManagerHighlighted.hasFunctionName}(value)) {
```

Comment:
To avoid extra texture fetches in the case this functionality isn't being used, the same conditional logic used for the equivalences table could also be used.